src/test-kit/instagram-clone-driver.ts

```diff
@@ -0,0 +1,72 @@
+import type { Locator, Page } from "@playwright/test";
+import {
+  COMMENTLIST,
+  COMMENTS,
+  FORM,
+  INPUT,
+  POSTCOMMENT,
+} from "./constants.js";
+
+export class InstagramCloneDriver {
+  private page: Page;
+  private inputLocator: Locator;
+  private postCommentLocator: Locator;
+  private commentsLocator: Locator;
+  private commentListLocator: Locator;
+  private formLocator: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.inputLocator = page.locator(INPUT);
+    this.postCommentLocator = page.locator(POSTCOMMENT);
+    this.commentListLocator = page.locator(COMMENTLIST);
+    this.formLocator = page.locator(FORM);
+    this.commentsLocator = page.locator(COMMENTS);
+  }
+
+  postComment = async (input: string) => {
+    await this.typeInput(input);
+    await this.page.keyboard.press("Enter");
+  };
+
+  async typeInput(input: string) {
```

Comment:
instead of awaiting in the driver method you can have it just return the promise.

it makes the syntax cleaner and doesn't make much difference to how this method is called